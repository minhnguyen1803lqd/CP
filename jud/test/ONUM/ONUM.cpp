#include <bits/stdc++.h>
using namespace std;
int main()
{
	long long a,b,n;
	freopen("onum.inp","r",stdin);
	freopen("onum.out","w",stdout);
	cin>>n>>a>>b;
	long long rest=(b*(n-1)+a)-(a*(n-1)+b)+1;
	if (a<=b) cout<<rest;
	else cout<<0;
	return 0;
}