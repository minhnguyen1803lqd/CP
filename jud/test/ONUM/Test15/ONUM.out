180913226545399537
